usecases/request.go

```diff
@@ -0,0 +1,41 @@
+package usecases
+
+import (
+	"github.com/gorilla/websocket"
+	"github.com/topfreegames/pitaya/client"
+)
+
+// RequestMsg ...
+type RequestMsg struct {
+	Route     string
+	Payload   []byte
+	IsRequest bool
+}
+
+// ProcessMessage ...
+func ProcessMessage(msg RequestMsg, pClient *client.Client) error {
+	var err error
+	if msg.IsRequest {
+		_, err = pClient.SendRequest(msg.Route, msg.Payload)
+	} else {
+		err = pClient.SendNotify(msg.Route, msg.Payload)
+	}
+	if err != nil {
```

Comment:
this check is useless since we only have two possibilities, you can return err directly